src/backend/api/library.ts

```diff
@@ -38,8 +30,16 @@ export const importGame = async (args: ImportGameArgs) =>
   ipcRenderer.invoke('importGame', args)
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const handleSetGameStatus = (callback: any) =>
+export const handleSetGameStatus = (callback: any) => {
   ipcRenderer.on('setGameStatus', callback)
+  return () => {
+    ipcRenderer.removeListener('setGameStatus', callback)
+  }
+}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+// export const setGameStatusRemoveListener = (onGameStatusUpdate: any) => {
+//   ipcRenderer.removeListener('setGameStatus', onGameStatusUpdate)
+// }
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const handleProgressOf = (version: string, callback: any) =>
   ipcRenderer.on('progressOf' + version, callback)
```

Comment:
This belongs to the wine-manager, not to the library. Feel free to rename the event to `'progressOfWineManager' + version`

Make sure also to adapt the sender. The callback is actually no callback i think. It is data. See the sender.

https://github.com/BrettCleary/HeroicGamesLauncher/blob/2d2e15b4dfa6c473a1e67e070e6a9e8822f14dbd/src/backend/wine/manager/ipc_handler.ts#L22-L36

Reply:
Noted on the naming and folder. On the last point, the sender sends `{state, progress}` to the callback, so the callback arguments for callback passed by the frontend are `(e: IpcRendererEvent, {state, progress})`